src/main/java/com/zaxxer/nuprocess/internal/BasePosixProcess.java

```diff
@@ -627,25 +616,20 @@ protected void callStart()
    protected void initFailureCleanup(int[] in, int[] out, int[] err)
    {
       Set<Integer> unique = new HashSet<>();
-      if (in != null) {
```

Comment:
The `in`, `out`, and `err` arrays can never be `null`; `createPipes` initializes all 3 before it makes the first native call.